### PR TITLE
278 camel case

### LINKED ISSUE #278: Ensure Usage of Consistent Naming
As a developer, I'd like the naming standard camelCase to be used consistently, in order to have code that is easy to read.

---

### :thought_balloon: &nbsp; Hints & Thoughts
There is one exception to this: We agreed on using PascalCase to name component files.
### :question: &nbsp; Additional Questions to Answer / Consider

---

## :white_check_mark: &nbsp; Conditions of satisfaction

- [x] Should check for all names that are not written in camelCase and rewrite them in camelCase (in Frontend, Backend, Local Client, Database)
- [x] Should make sure that the naming conventions of file naming are used consistently (Frontend, Backend, Local Client)
- [x] Should check that changes to the code and database have not effected the behavior of the product
- [x] rename all workflow (camelCase) and wiki (Title-Case) files consistently

### PR DESCRIPTION
**Pull request by SanJSp**
Fixes #278 

## Changes
This PR enforces uniform naming of our files and code in camelCase or if specified in our coding standards otherwise. Furthermore, I've adapted the liveLogsListener in the local client, so please also review that here bptlab/ark_automate_local#7 :) 

The changes to the DB will be done as soon as this PR is approved :)

## Checklist before merge

**Developer's responsibilities**
* [x] **Assign** one or two developers
* [x] **Change code** if reviewer(s) has/have requested it
* [x] **Pull request build** has passed
* [x] **tested locally** (in at least chrome & firefox if frontend)
* [x] updated the **documentation**
* [x] added **tests** where necessary

### CODE REVIEW

File: server/api/models/rpaTaskModel.js [hunk 1]
```diff
@@ -11,9 +11,9 @@ const rpaParameterSchema = new Schema({
 });
 
 const rpaTaskSchema = new Schema({
-  Application: { type: String, required: [true, 'Application required'] },
-  Task: { type: String, required: [true, 'Task required'] },
-  Code: { type: String, required: [true, 'Code required'] },
+  application: { type: String, required: [true, 'Application required'] },
```
Comment (kej-jay) [inline server/api/models/rpaTaskModel.js:14]: Will this still work with the existing data in the database?

**Review by kej-jay — COMMENTED** [review]
Great job. Please address my one question regarding the attribute names in MongoDB, other than that I will approve

File: server/api/models/rpaTaskModel.js (hunk 1 above)
Comment (SanJSp) [reply server/api/models/rpaTaskModel.js:14]: I will change all the data in the database 😁

File: server/api/models/rpaTaskModel.js [hunk 2]
```diff
@@ -11,9 +11,9 @@ const rpaParameterSchema = new Schema({
...
+  task: { type: String, required: [true, 'Task required'] },
+  code: { type: String, required: [true, 'Code required'] },
   outputValue: Boolean,
   inputVars: [rpaParameterSchema],
   Output: rpaParameterSchema,
```
Comment (WolfgangDaniel) [inline server/api/models/rpaTaskModel.js:19]: Output should also be written small: "output". Also change all occurences of this Output property in the code.

File: server/socket/socketManager.js [hunk 3]
```diff
@@ -78,16 +78,16 @@ exports.socketManager = (io, socket) => {
```
Comment (WolfgangDaniel) [inline server/socket/socketManager.js:78]: Oddly enough, I can't comment on lines 27 and 29. This comment is for line 27 and line 29. Please change there robot_id to robotId

**Review by WolfgangDaniel — CHANGES_REQUESTED** [review]
Just some minor changes. Once they are changed I will approve :)

File: server/socket/socketManager.js (hunk 3 above)
Comment (SanJSp) [reply server/socket/socketManager.js:78]: well spotted!

**Review by WolfgangDaniel — APPROVED** [review]
Thanks for implementing the changes. It looks good :D Please make sure to test the application carefully after you changed the objects in the database.

**Review by kej-jay — APPROVED** [review]
I will approve this with the hope that you take care of restructuring everything in the database accordingly.
For the next time I would have preferred a database with these changes to which I could connect my local backend, so that I could actually test if the software is still working properly.

**Pull request merged**